### PR TITLE
Oli 170711 02

### PR DESCRIPTION
**Pull request by gutsche**
created nominal disk and tape model relying on MINIAOD. Now questioning how we include HL-LHC MC in the calculations.

### CODE REVIEW

**Comment by kenbloom** [comment]
@ericvaandering Are you OK with these changes?

File: data.py [hunk 1]
```diff
@@ -80,13 +80,15 @@
...
 
+
 diskCopies = {}
 tapeCopies = {}
 for tier in TIERS:
     diskCopies[tier] = [versions * replicas for versions, replicas in
                         zip(model['storage_model']['versions'][tier], model['storage_model']['disk_replicas'][tier])]
     # Assume we have the highest number of versions in year 1, save n replicas of that
     tapeCopies[tier] = model['storage_model']['versions'][tier][0] * model['storage_model']['tape_replicas'][tier]
+    if len(tapeCopies[tier]) == 0: tapeCopies[tier] = [0,0,0]
```
Comment (ericvaandering) [inline data.py]: Please do 

'''if not tapeCopies[tier]:
tapeCopies[tier] = [0,0,0]'''

File: data.py [hunk 2]
```diff
@@ -178,16 +180,26 @@
...
+diskByTier[YEARS.index(2017)][TierColumns.index('Run1 & 2')] = 25
+diskByTier[YEARS.index(2018)][TierColumns.index('Run1 & 2')] = 10
+diskByTier[YEARS.index(2019)][TierColumns.index('Run1 & 2')] = 5
+diskByTier[YEARS.index(2020)][TierColumns.index('Run1 & 2')] = 0
+
+diskByYear[YEARS.index(2017)][YearColumns.index('Run1 & 2')] = 25
+diskByYear[YEARS.index(2018)][YearColumns.index('Run1 & 2')] = 10
+diskByYear[YEARS.index(2019)][YearColumns.index('Run1 & 2')] = 5
+diskByYear[YEARS.index(2020)][YearColumns.index('Run1 & 2')] = 0
+
```
Comment (ericvaandering) [inline data.py:192]: Yeah, I don't like this. We should revisit after we produce the document. I. worked hard to not have anything of our model aside from MC/data in my code and I want to keep it that way.

File: data.py (hunk 1 above)
Comment (gutsche) [reply data.py]: Done.

**Pull request merged**